src/Bicep.Core.IntegrationTests/RegistryProviderTests.cs

```diff
@@ -149,8 +149,7 @@ public async Task Third_party_imports_are_disabled_unless_feature_is_enabled()
         var result2 = await CompilationHelper.RestoreAndCompile(service, @$"
 provider 'br:example.azurecr.io/test/provider/http@1.2.3'
 ");
-        result2.Should().HaveDiagnostics(new[] {
```

Comment:
Ideally, I was hoping to avoid hitting the ThirdParty logic unless the ProviderRegistry feature flag was enabled.

But since we do not want to check the feature flag itself in the DefaultNamespaceProvider, the "http@123" is still recognized even though the feature flag is not enabled.

Reply:
Is this okay?

Reply:
Decided to use the ProviderRegistry feature flag check in DefaultNamespace